gvm/protocols/gmpv8/gmpv8.py

```diff
@@ -1089,16 +1040,18 @@ def modify_schedule(
         cmd = XmlCommand("modify_schedule")
         cmd.set_attribute("schedule_id", schedule_id)
 
-        if name:
-            cmd.add_element("name", name)
+        if name != GvmProtocol._UNDEFINED_VALUE:
```

Comment:
Same. Isn't this something for `_check_input`?

Reply:
I will create some other PRs and split things up ...